agent-core/src/main/java/com/newrelic/agent/android/instrumentation/TransactionState.java

```diff
@@ -229,7 +240,7 @@ TransactionData toTransactionData() {
                     bytesSent,
                     bytesReceived,
                     appData,
-                    wanType, trace, null);
+                    wanType, trace,"",params, null);
```

Comment:
Formatting. Don't forget to clean before commit. What does the parameter `""` represent? Is there a better way to communicate this (such as using a string constant)?

Reply:
this responsebody value is set by setter method and Transcation state does not have response body, i am passing empty value . we can add constant value for empty string.